components/sendloop/actions/add-subscriber/add-subscriber.mjs

```diff
@@ -0,0 +1,42 @@
+import sendloop from "../../sendloop.app.mjs";
+
+export default {
+  key: "sendloop-add-subscriber",
+  name: "Add Subscriber",
+  description: "Adds a new subscriber to a specified list.",
```

Comment:
Do we have API documentation links for these components?

Reply:
You have to be logged into the UI to see the docs. 😦